[2024-04-28T15:55:13.248+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-28T15:55:13.265+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Final_exam.insert_db scheduled__2024-04-28T15:52:04.190626+00:00 [queued]>
[2024-04-28T15:55:13.271+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Final_exam.insert_db scheduled__2024-04-28T15:52:04.190626+00:00 [queued]>
[2024-04-28T15:55:13.271+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-04-28T15:55:13.283+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): insert_db> on 2024-04-28 15:52:04.190626+00:00
[2024-04-28T15:55:13.290+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=161) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-28T15:55:13.292+0000] {standard_task_runner.py:63} INFO - Started process 163 to run task
[2024-04-28T15:55:13.291+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Final_exam', 'insert_db', 'scheduled__2024-04-28T15:52:04.190626+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/Project.py', '--cfg-path', '/tmp/tmpvdvkrf33']
[2024-04-28T15:55:13.292+0000] {standard_task_runner.py:91} INFO - Job 13: Subtask insert_db
[2024-04-28T15:55:13.328+0000] {task_command.py:426} INFO - Running <TaskInstance: Final_exam.insert_db scheduled__2024-04-28T15:52:04.190626+00:00 [running]> on host 73ada3aeb297
[2024-04-28T15:55:13.387+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ha20040204@gmail.com' AIRFLOW_CTX_DAG_OWNER='HA' AIRFLOW_CTX_DAG_ID='Final_exam' AIRFLOW_CTX_TASK_ID='insert_db' AIRFLOW_CTX_EXECUTION_DATE='2024-04-28T15:52:04.190626+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-28T15:52:04.190626+00:00'
[2024-04-28T15:55:13.388+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-28T15:56:06.078+0000] {logging_mixin.py:188} INFO - Successfully.
[2024-04-28T15:56:06.087+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-28T15:56:06.088+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-28T15:56:06.094+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=Final_exam, task_id=insert_db, execution_date=20240428T155204, start_date=20240428T155513, end_date=20240428T155606
[2024-04-28T15:56:06.122+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-28T15:56:06.148+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-28T15:56:06.150+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
